[2022-12-25T19:42:51.666+0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_pivot.first_task scheduled__2022-07-28T15:00:00+00:00 [queued]>
[2022-12-25T19:42:51.676+0300] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: titanic_pivot.first_task scheduled__2022-07-28T15:00:00+00:00 [queued]>
[2022-12-25T19:42:51.677+0300] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-25T19:42:51.677+0300] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2022-12-25T19:42:51.677+0300] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-25T19:42:51.693+0300] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): first_task> on 2022-07-28 15:00:00+00:00
[2022-12-25T19:42:51.698+0300] {standard_task_runner.py:55} INFO - Started process 29262 to run task
[2022-12-25T19:42:51.706+0300] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'titanic_pivot', 'first_task', 'scheduled__2022-07-28T15:00:00+00:00', '--job-id', '482', '--raw', '--subdir', 'DAGS_FOLDER/first_dag_total.py', '--cfg-path', '/var/folders/g7/wpcjf399379f4cptqxrgszxr0000gn/T/tmpg2ajbpw9']
[2022-12-25T19:42:51.709+0300] {standard_task_runner.py:83} INFO - Job 482: Subtask first_task
[2022-12-25T19:42:51.733+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:42:51.735+0300] {logging_mixin.py:137} WARNING - /Users/vadim/opt/anaconda3/lib/python3.9/configparser.py:408 FutureWarning: section/key [core/sql_alchemy_conn] has been deprecated, you should use[database/sql_alchemy_conn] instead. Please update your `conf.get*` call to use the new name
[2022-12-25T19:42:51.737+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/settings.py:400 DeprecationWarning: The sql_engine_encoding option in [core] has been moved to the sql_engine_encoding option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:42:51.739+0300] {logging_mixin.py:137} WARNING - /Users/vadim/opt/anaconda3/lib/python3.9/configparser.py:408 FutureWarning: section/key [core/sql_engine_encoding] has been deprecated, you should use[database/sql_engine_encoding] instead. Please update your `conf.get*` call to use the new name
[2022-12-25T19:42:51.770+0300] {logging_mixin.py:137} WARNING - /Users/vadim/.venv_***/lib/python3.9/site-packages/***/utils/sqlalchemy.py:121 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2022-12-25T19:42:51.814+0300] {task_command.py:389} INFO - Running <TaskInstance: titanic_pivot.first_task scheduled__2022-07-28T15:00:00+00:00 [running]> on host mbp-vadim.bbrouter
[2022-12-25T19:42:51.911+0300] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=titanic_pivot
AIRFLOW_CTX_TASK_ID=first_task
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T15:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-28T15:00:00+00:00
[2022-12-25T19:42:51.912+0300] {subprocess.py:63} INFO - Tmp dir root location: 
 /var/folders/g7/wpcjf399379f4cptqxrgszxr0000gn/T
[2022-12-25T19:42:51.913+0300] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "Here we start!"']
[2022-12-25T19:42:51.924+0300] {subprocess.py:86} INFO - Output:
[2022-12-25T19:42:51.927+0300] {subprocess.py:93} INFO - Here we start!
[2022-12-25T19:42:51.928+0300] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-25T19:42:51.967+0300] {taskinstance.py:1322} INFO - Marking task as SUCCESS. dag_id=titanic_pivot, task_id=first_task, execution_date=20220728T150000, start_date=20221225T164251, end_date=20221225T164251
[2022-12-25T19:42:52.003+0300] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-25T19:42:52.045+0300] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
